pkg/option/config.go

```diff
@@ -912,6 +912,10 @@ const (
 	// EnableCustomCallsName is the name of the option to enable tail calls
 	// for user-defined custom eBPF programs.
 	EnableCustomCallsName = "enable-custom-calls"
+
+	// EnableEgressRevnatAtHostOnly enables reverse NAT translation at node's
+	// host interface instead of veth interface on egress datapath.
+	EnableEgressRevnatAtHostOnly = "enable-egress-revnat-at-host-only"
```

Comment:
I would name it `--bpf-revnat-at-host` given it's a bpf internal detail. We typically prefix those with `bpf-*`.

Reply:
Also, I wonder to make sure we catch any unintentional regressions, should we add a CI test?

Reply:
^ Verifier complexity with the explosion of options is death by a thousand cuts since any combination of the datapath options needs to be uniquely configured & tested. Is it plausible to make this part of the default behaviour?

Reply:
> Is it plausible to make this part of the default behaviour?

Unfortunately not, since this would cause performance regression for nodeport traffic, since the skb would need to go up the stack, get orphaned before hitting routing layer, and give TCP non-optimal back pressure signal.